cs.html

```diff
@@ -1632,6 +1693,22 @@ <h3>Unreliable delivery</h3>
 }
 </pre>
  </section>
+
+ <section class="informative" id="datagramexamples*">
+    <h3>Sending QUIC Datagrams</h3>
+    <p>Sending QUIC datagrams can be achieved by using the <code>sendDatagram</code>
+    and <code>readyToSendDatagram</code> methods.</p>
+<pre class="example highlight">
+let quic = getQuicTransport();
+let packets = getPacketsToSend();
+packets.forEach((packet) => {
+  quic.readyToSendDatagram().then(() => {
+    sendDatagram(packet);
+  });
```

Comment:
Might be good to show another example that doesn't use quic.readyToSendDatagram, perhaps sending at a particular rate (once per 100ms?)

Reply:
Were you thinking something simple like this?
```javascript
const quic = getQuicTransport();
setInterval(() => {
  sendDatagram(currentPacket());
}, 100);
```

Reply:
Yes, except "quic.sendDatagram" and perhaps "createDatagram()" instead of "currentPacket()"?

Reply:
sgtm. I made this fix. On a side note what do you think of examples for more complex scenarios? For example sending a datagram based upon some event (perhaps an event in a game?). For example:
```javascript
const quic = getQuicTransport();
function OnDatagramEvent(datagram) {
  const eventTime = Date.now();
  quic.readyToSendDatagram().then(() => {
    // This will immediately resolve if the transport is not blocked
    // by congestion control or connection level flow control. If it
    // was blocked we only want to send the datagram if it became unblocked
    // within 100 ms and the datagram is still worth sending.
    const sendTime = Date.now();
    if (sendTime - eventTime > 100) {
      quic.sendDatagram(packet).then((acked) => {
        if (acked) {
          console.log('datagram ' + datagram.id + ' was acked');
        } else {
          console.log('datagram ' + datagram.id + ' was lost');
        }
      });
    }
  });
}
```